assets/sass/config/_mixins.scss

```diff
@@ -25,6 +25,10 @@
 	}
 }
 
+@mixin elevation-3 {
+	box-shadow: 0 1.25px 5px 0 rgba(0, 0, 0, 0.19), 0 0.3333333134651184px 1.5px 0 rgba(0, 0, 0, 0.039);
```

Comment:
Where does this elevation mixin come from? Just curious as to the `0.3333333134651184px` value, I am guessing that's been copied in from somewhere?

Reply:
Yep, that's what was copied in from Figma, so I'm also not sure why it's so specific 😄 

It could probably be rounded a bit.

Reply:
Ah, if it's come from Figma then maybe we should round it - I was guessing it might have come from some Material SASS/CSS in which case I could see the argument for keeping it... (although it'd still be a bit of an odd number to be fair).